packages/rabbitmq-server/build.sh

```diff
@@ -0,0 +1,46 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/rabbitmq/rabbitmq-server
+TERMUX_PKG_DESCRIPTION="Feature rich, multi-protocol messaging and streaming broker"
+TERMUX_PKG_LICENSE="MPL-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="4.0.3"
+TERMUX_PKG_SRCURL=https://github.com/rabbitmq/rabbitmq-server/releases/download/v${TERMUX_PKG_VERSION}/rabbitmq-server-generic-unix-${TERMUX_PKG_VERSION}.tar.xz
+TERMUX_PKG_SHA256=607ba3b4fd1fc133f3bada1b2225ff0527f99cc6eb32423ae73f6c9f5ee9bd13
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_HOSTBUILD=true
```

Comment:
Why are we `HOSTBUILD`-ing anyway?
You overwrite the hostbuild step with a no-op down below anyway, so we aren't actually doing a hostbuild.

Might be a misunderstanding of what the host build actually is.

Reply:
Host builds can be a bit difficult to wrap your head around.
They're not usually required for most packages.

It's a build of the package for the platform ***hosting*** the build system, for example our package build container.
This is sometimes necessary to generate certain parts of packages that cannot be cross-compiled properly.